packages/core/src/KeyMapping.ts

```diff
@@ -0,0 +1,78 @@
+import { defaultKeyMap } from './DefaultKeyMap';
+
+export interface Keypress {
+    key: string;
+    ctrlKey?: boolean;
+    altKey?: boolean;
+    shiftKey?: boolean;
+}
+export interface CommandConfig {
+    name: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    arguments: Record<string, any>;
+}
+export type ShortcutIdentifier = string; // format: /[Ctrl+][Alt+][Shift+]key/
+export type KeyMap = Map<ShortcutIdentifier, CommandConfig>;
+
+export class KeyMapping {
```

Comment:
Do we really need this class ? Considering the only component using the keymap is the event manager, I would just give a `Map<ShortcutIdentifier, CommandConfig>` to the event manager, give it a `matchShortcut` and `setShortcut` functions and be done with it, to reduce intermediary concepts.